README.md

```diff
@@ -1 +1,48 @@
-# Data Science Stack
+<div align="center">
+
+# Data Science Stack ✨
+
+*Making it seamless to run GPU enabled containerized ML Environments*
+
+</div>
```

Comment:
We don't do that in other repos. It's fine but it's a bit out of the style we have.

Reply:
I've seen a similar README in the [oci-factory](https://github.com/canonical/oci-factory) and thus the sparkle emoji and the difference.

I'd personally like for our products (Kubeflow, DSS, MLFlow) to start have more "flashy" docs, compared to for example our charm repos.

Of course this will become a more focused effort once we also get a technical author. @orfeas-k if you don't have a hard preference against it I'd suggest we keep it like this for now and we can revisit later on once we'll be finalising the README

Reply:
SGTM